plugins/catalog-backend-module-gitlab/src/lib/client.ts

```diff
@@ -136,16 +236,26 @@ export class GitLabClient {
         continue;
       }
 
-      memberIds.push(
-        ...response.data.group.groupMembers.nodes
-          .filter(n => n.user)
-          .map(node =>
-            Number(node.user.id.replace(/^gid:\/\/gitlab\/User\//, '')),
-          ),
-      );
+      const usersData = response.data.group.groupMembers.nodes;
+
+      for (let i = 0; i < usersData.length; i++) {
+        const userItem = usersData[i];
```

Comment:
Could also have used just a `for (const userItem of response.data.group.groupMembers.nodes) {` (same above), saving two variables

Reply:
Nice, I'll take a look at adopting this approach, thanks!